composer.json

```diff
@@ -17,7 +17,8 @@
 		"kdyby/events": "~2.4",
 		"nette/bootstrap": "~2.3",
 		"nette/application": "~2.3",
-		"tracy/tracy": "~2.3"
+		"tracy/tracy": "~2.3",
+		"phpunit/phpunit": "^5.0"
```

Comment:
I never do that with unit testing tools. It just doesn't feel right.

Your tests require PHPUnit and therefore PHPUnit is a dev dependency of this package, there really is nothing to discuss about it. All sane packages with phpunit test have phpunit as dev dependency and you should do the same.